pwndbg/commands/cymbol.py

```diff
@@ -141,6 +141,24 @@ def add_custom_structure(custom_structure_name: str) -> None:
     load_custom_structure.__wrapped__(custom_structure_name, pwndbg_custom_structure_path)
 
 
+def add_custom_structure_from_file(header_file_path: str) -> None:
+    with open(header_file_path, "r") as file:
+        custom_structures_source = file.read().strip()
+
+    if not custom_structures_source:
+        print(message.notice("The header file is empty or does not contain valid structures."))
+        return
+
+    custom_structure_name = os.path.splitext(os.path.basename(header_file_path))[0]
+    pwndbg_custom_structure_path = os.path.join(pwndbg_cachedir, custom_structure_name) + ".c"
+
+    with open(pwndbg_custom_structure_path, "w") as f:
+        f.write(custom_structures_source)
```

Comment:
What is this exactly for / why do we need to write the file back again?

Reply:
it was to allow for user's custom structures to persist for reuse across sessions, but if this is unneeded i'll toss it